docs/_source/community/developer_docs.md

```diff
@@ -203,58 +231,84 @@ See https://github.com/opensearch-project/k-NN/issues/1286
 This may result in unexpected results when combining filtering with vector search with this engine.
 :::
 
-### Relational Database and Migration
+#### Relational Database and Migration
 
 Argilla will use SQLite as the default built-in option to store information about users, workspaces, etc. for the relational database. No additional configuration is required to start using SQLite.
 
 By default, the database file will be created at `~/.argilla/argilla.db`, this can be configured by setting different values for `ARGILLA_DATABASE_URL` and `ARGILLA_HOME_PATH` environment variables.
 
-#### Run Database Migration
+##### Create the Default User
```

Comment:
I believe we should run migrations before the user creation

Reply:
and maybe this segment is duplicated?

Reply:
weirdly, it was duplicated before and after. I left run migrations before the user creation